### PR TITLE
Build and release binary on CI

### PR DESCRIPTION
**Pull request by tomyrd**
After pushing a tag the CI will build the binary and release it. This released binary can be used in the servers instead of compiling each time.

### CODE REVIEW

File: .github/workflows/binary-release.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      - name: Install Ignite
+        run: curl https://get.ignite.com/cli! | bash
+      - name: Build binary
+        run: ignite chain build -o ./
+      - name: Create release
+        uses: softprops/action-gh-release@v2
+        if: startsWith(github.ref, 'refs/tags/')
+        with:
+          files: alignedlayerd
+          prerelease: true
```
Comment (MauroToscano) [inline .github/workflows/binary-release.yml:28]:
```suggestion
          prerelease: true

```

**Review by MauroToscano — APPROVED** [review]

**Review by JulianGCalderon — DISMISSED** [review]
LGTM

**Review by ManuelBilbao — CHANGES_REQUESTED** [review]
This action will trigger on every push. This seems a bit overwhelm. I think you can trigger it only on push tags

**Review by JulianGCalderon — APPROVED** [review]
LGTM

**Review by ManuelBilbao — APPROVED** [review]

**Pull request merged**